src/Message.php

```diff
@@ -314,6 +314,10 @@ public function setSender($emailOrAddress, $name = null)
      */
     public function getSender()
     {
+        $headers = $this->getHeaders();
```

Comment:
Calling `Message::getHeaderByName` will create a new blank header if the requested header does not exist, as per the docbloc.

The existing strategy as per `Message::getSubject` is to check if the header exists before calling `getHeaderByName`. (The existing code also uses a local variable for $headers.)